frame/contracts/src/wasm/runtime.rs

```diff
@@ -1847,4 +1877,44 @@ define_env!(Env, <E: Ext>,
 			Err(_) => Ok(ReturnCode::EcdsaRecoverFailed),
 		}
 	},
+
+	// Execute code in the context of the current contract.
+	//
+	// # Parameters
+	//
+	// - flags: See [`CallFlags`] for a documenation of the supported flags.
+	// - code_hash: a pointer to the address of the code to be called.
+	// - input_data_ptr: a pointer to a buffer to be used as input data to the callee.
+	// - input_data_len: length of the input data buffer.
+	// - output_ptr: a pointer where the output buffer is copied to.
+	// - output_len_ptr: in-out pointer to where the length of the buffer is read from
+	//   and the actual length is written to.
+	//
+	// # Errors
+	//
+	// An error means that the call wasn't successful output buffer is returned unless
+	// stated otherwise.
+	//
+	// `ReturnCode::CalleeReverted`: Output buffer is returned.
+	// `ReturnCode::CalleeTrapped`
```

Comment:
Is that possible to return an error that the code does not exist?

Reply:
that would be a dispatch error `CodeNotFound`

Reply:
`CodeNotFound` is returned
thanks for pointing out